src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -176,8 +176,8 @@
                       i18n>
                       Use a dedicated data pool
 
-          <cd-help-text>Use a dedicated pool to store the mirror data. If not selected,
-            the mirror data will be stored in the same pool as the image data.
+          <cd-help-text>Use a dedicated pool to store the data. If not selected,
+            the data will be stored in the same pool as the image data.
```

Comment:
"[...] the data will be stored in the same pool as the image data" is tautological
```suggestion
          <cd-help-text>Use a dedicated pool to store the image data. If not selected,
            the image data will be stored in the same pool as the image metadata.
```

Reply:
Thanks @idryomov , Updated